index.php

```diff
@@ -397,9 +352,9 @@ class="btn-large waves-effect waves-light light blue lighten-1" style="border-ra
 	<br>
 	<h4 class="header center light blue-text text-lighten-2">Featured Projects</h4>
 	<h6 class="header center light text-shadow: 2px 2px 7px #111111" style="padding:0% 10% 2% 10%">(100% completion rate)</h6>
-	<div class="icon-block" style="width:100%"><i class='material-icons left' style="font-size:20px;color:#03A9F4">timeline</i> = &nbsp;village data trends available
+	<!--  <div class="icon-block" style="width:100%"><i class='material-icons left' style="font-size:20px;color:#03A9F4">timeline</i> = &nbsp;village data trends available
 	<br>
-	<i class='material-icons left' style="font-size:20px;color:#03A9F4">fiber_new</i> = &nbsp;data trends coming soon</div>
+	<i class='material-icons left' style="font-size:20px;color:#03A9F4">fiber_new</i> = &nbsp;data trends coming soon</div> -->
```

Comment:
In general, version control works a lot better if you remove blocks instead of commenting them out.